kafka-streams-quickstart/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.5'
 
 services:
   zookeeper:
-    image: strimzi/kafka:0.11.3-kafka-2.1.0
+    image: strimzi/kafka:0.19.0-kafka-2.5.0
```

Comment:
We should update all our docker-compose files.

Reply:
I couldn't find any other references to "kafka:0.11.3-kafka-2.1.0" in the quickstart repo? DYM elsewhere? I'll send a PR for updating some references I found in docs.

Reply:
I was thinking about:
* https://github.com/quarkusio/quarkus-quickstarts/blob/development/kafka-quickstart/docker-compose.yaml
* https://github.com/quarkusio/quarkus-quickstarts/blob/development/kafka-panache-quickstart/docker-compose.yaml

But no worries, I will open a PR about these.